src/lib/OpenEXR/ImfRgbaFile.h

```diff
@@ -320,6 +320,10 @@ class RgbaInputFile
     RgbaInputFile (const char name[], int numThreads = globalThreadCount());
 
 
+    IMF_EXPORT
+    RgbaInputFile (int partNumber, const char name[], int numThreads = globalThreadCount());
```

Comment:
I think it's a great addition, and I like your approach. If you'd like to go ahead and fix the DCO and CLA issues, we can merge it as is. I'm happy to make extra tweaks on top of that. Otherwise, I guess we can start again?

Reply:
I'd say take the code and run. I could go about the DCO/CLA stuff but if it's OK with you just take the commit, cherry pick or copy or use a basis, as you prefer. From my perspective I proposed this because I need this in a project, and it's not convenient to maintain a fork.
Let me know what you think.

Reply:
Technically, that's against the spirit behind the DCO/CLA. The PR is your code, so "take it and run" is really just another way of saying "commit it without a DCO or CLA," right? Somebody else re-typing it doesn't change the original authorship or ownership of the code. It's the DCO/CLA that is the formality that certifies to us that it is indeed ok with the owner of the code to incorporate it into the project under the project's license terms.

Reply:
DCO/CLA done. Now you can legally take the code and run ;)